### PR TITLE
Add support for linux brew (cont.)

### PR DESCRIPTION
**Pull request by jmccann**
Continuation of #15

### CODE REVIEW

File: Abstract/abstract-drone.rb [hunk 1]
```diff
@@ -65,7 +65,30 @@ def self.latest_sha256sum
       raise "Could not find checksum"
     end
     url = checksum_assest.first['browser_download_url']
-    `curl -L -s #{url}`.lines.grep(/drone_darwin_amd64.tar.gz/).first.split(' ').first
+    `curl -L -s #{url}`.lines.grep(/#{drone_file_name}/).first.split(' ').first
```
Comment (tboerger) [inline Abstract/abstract-drone.rb]: download_file?

**Review by tboerger — COMMENTED** [review]
Maybe just add a helper linux? Instead of os?

File: Abstract/abstract-drone.rb (hunk 1 above)
Comment (jmccann) [reply Abstract/abstract-drone.rb]: fixed, thanks for the review

**Comment by jmccann** [comment]
I admit I only need to detect linux.  I also admit that was a copy/paste for os detection.

Didn't know if it was worth keeping.  tbh, I didn't even know there was linuxbrew.  Who knows if/when someone tries to port to another OS?

I can refactor that if you think it's better to simplify it.  I don't care much either way. 😛

**Review by tboerger — APPROVED** [review]

**Pull request merged**